contents/docs/user-guides/funnels.md

```diff
@@ -4,74 +4,175 @@ sidebar: Docs
 
-Once you're on the 'Funnels' page, you can then start to create your funnel.
+<img width="1360" alt="image" src="https://user-images.githubusercontent.com/85295485/134478357-d99d48ae-0d59-4f40-829b-b6f19c53ed0e.png">
 
-Click on the blue 'Add action/event' button and select the action or event you want to use as the starting point for your funnel.
+**Step 2: Add and refine steps in your funnel**
 
-You can then add a filter to that specific action now or a general filter to the entire funnel later on ('Filters' -> 'Add filter'). 
+Select the steps you wish to include in your funnel, ideally starting with the first event/action a user will trigger in the flow you wish to measure and then add intermediate events you believe they would need to go through to be successful and finish with the event which you consider to be the success for this flow, start with the simplest flow and avoid using optional steps to ensure you don’t filter out or skew valuable results.
 
-It's also worth noting how the PostHog funnels work in terms of what counts as a conversion to the next step. When your funnel is ready, you will have the ability to specify a time range for your funnel, such as "Last 7 days", "Last 48 hours", or a custom range. 
+![image](https://user-images.githubusercontent.com/85295485/134485100-23d02b5b-c3ad-4704-90d9-fc3d4db93c39.png)
 
-Then, based on that time range, PostHog will determine conversion based on if the user performed one action after the other _at any point during the time period_. That means that if my time range is set to 2 weeks, a user may perform step 1 on day 2 and step 2 on day 13 and it will still count as a conversion. 
+You may wish to refine your steps further, filtering out certain types of users or event properties (e.g. by country of a user), click the “filter” icon next to the step you wish to refine further and select the filter you would like to add or set a global filter under the filters section on the right.
 
-### Step 3: Calculating funnel
+<img width="442" alt="image" src="https://user-images.githubusercontent.com/85295485/134478622-df444540-7962-436e-bb86-98481b8563ab.png">
 
-Once you're satisfied with all the steps you've added, you can then click 'Calculate' to generate the funnel.
+**Step 3: Set your time range**
 
-### Filter funnels by time
+You should now have a basic funnel showing in the main analysis pane, you can now modify the time range for which you would like to look at the funnel. A good place to start is to look at a time range of 30 days to ensure the effects of any day to day variations will be reduced.
 
-You can also filter funnels by time intervals. By default, funnels will show actions completed over the last 7 days, but this can be changed by selecting the time interval next to 'Add to dashboard'
+![image](https://user-images.githubusercontent.com/85295485/134479552-fa5a3eeb-762e-4a01-b5ab-01c43877f120.png)
```

Comment:
Think it might be better to host images directly on the repo (or S3 alternatively), (see https://github.com/PostHog/posthog.com/issues/2071), but perhaps not the best idea to reference to GitHub, there's no guarantee the images will stay there.